[BLOG] Add low-code-workshop (#515)
Quansight/Quansight-website: apps/labs/posts/low-code-workshop.md

---
title: 'Low-code contributions through GitHub'
published: September 28, 2021
author: isabela-presedo-floyd

> description: "Our goal with the following low-code workshop is to offer a way for folks to join a project's contributors list without the technical overhead. To achieve this we'll discuss a format that relies solely on the GitHub web interface."

Reviewer: ```suggestion
description: "Low-code contribution workshops offer a way for folks to join a project's contributors list without the technical overhead. In this blog we present a workshop format that relies solely on the GitHub web interface."
```

submit the pull request to the mainline project and see the work accepted into
the code base.

## Success stories

Everyone loves an underdog story.  What follows are there triumphs of some open
source allies who thought they could run an hour sprint that allowed multiple
folks to contribute to an open source project.  Believe it or not, we weren't
lying to ourselves and others. Below we highlight our successes with running
low-code sprints in Jupyter and NumPy.

### Jupyter accessibility group

There is a small group of the Jupyter community focused on the improving the
accessibility of JupyterLab and orbiting projects.  We meet every other work to
add small patches and fixes. Currently, we're in the process of organizing
workshops to advocate for accessibility and improve the quality of the entire
Jupyter experience for disabled scientists.

On an off week, between our normal syncs, we tested a low-code format for folks
to participate in the Jupyter project.  During an hour, four people were
included in the project by suggesting changes to alt text for images in the
JupyterLab documentation (https://github.com/isabela-pf/jupyterlab/pull/1),

> resulting in a collaborative commit to mainline with the help of @krassowski

Reviewer: ```suggestion
resulting in a collaborative commit to mainline with the help of [@krassowski](https://github.com/krassowski)
```

> and @isabela-pf as champions for the shared

Reviewer: ```suggestion
and [@isabela-pf](https://github.com/isabela-pf) as champions for the shared
```
